packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.test.js

```diff
@@ -560,18 +561,24 @@ const MOCK_PRODUCT = {
     c_width: 'Z'
 }
 
-test('Renders AddToCartModal', () => {
+test('Renders AddToCartModal with multiple products', () => {
     const MOCK_DATA = {
         product: MOCK_PRODUCT,
         itemsAdded: [
             {
                 product: MOCK_PRODUCT,
                 variant: MOCK_PRODUCT.variants[0],
                 quantity: 22
+            },
+            {
+                product: MOCK_PRODUCT,
+                variant: MOCK_PRODUCT.variants[0],
+                quantity: 1
             }
         ]
     }
-    const {getByText} = renderWithProviders(
+
+    renderWithProviders(
```

Comment:
I would `await renderWithProviders(`

Reply:
Our `renderWithProviders`, which would call `render` from the testing library, is not an async method. It returns an object. So calling `await` here would not do anything.

![iTerm2 2023-02-27 at 13 43 50](https://user-images.githubusercontent.com/847300/221692742-7a611f15-a811-4c01-b7eb-ba6153876e4c.png)